spec/system/accessibility_spec.rb

```diff
@@ -88,12 +88,12 @@
     end
 
     it 'workload report page has no axe detectible accessibility issues' do
-      visit report_path(:workload_report)
+      visit reporting_report_path(:workload_report)
       expect(page).to be_axe_clean.according_to accessibility_standards
     end
 
     it 'processed report page has no axe detectible accessibility issues' do
```

Comment:
I know this is not you but given we are here in the file can we change detectible --> detectable :)